api/src/main/java/io/opencensus/trace/unsafe/CtxUtils.java

```diff
@@ -0,0 +1,46 @@
+package io.opencensus.trace.unsafe;
+
+import io.opencensus.trace.ContextManager;
+import io.opencensus.trace.Ctx;
+import io.opencensus.trace.Span;
+
+public class CtxUtils {
+  // No instance of this class.
+  private CtxUtils() {}
+
+  private static final ContextManager DEFAULT_CONTEXT_MANAGER = new ContextManagerImpl();
+  private static ContextManager contextManager = DEFAULT_CONTEXT_MANAGER;
+
+  /**
+   * Overrides context manager with a custom implementation
+   * @param cm custom {@code ContextManager} to be used instead of a default one.
+   */
+  public static void setContextManager(ContextManager cm) {
```

Comment:
This is the key change in the OpenCensus Java code, allowing us to set a custom implementation, which can be hosted in the `opentelemetry-operations-java` repo. This way we don't need to add a dependency on OpenTelemetry jars here.

The implementation may be customized to make it thread safe later, keeping it simple for now.